modules/container/src/test/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,3 @@
+module.name=test-container
+
+artifact.export.classPackages=org.mule.runtime.container
```

Comment:
There is no such package "org.mule.runtime.container". Is this file really needed?